### ISSUE TITLE
Blank screen on Windows 11

### ISSUE THREAD

**Issue by TheComputer314** [desktop/desktop#16190]
### The problem

After updating the beta client to 3.1.9-beta1, the client only shows a blank white window that does nothing.

### Release version

3.1.9-beta1

### Operating system

Windows 11

### Steps to reproduce the behavior

1. Update the beta client from 3.1.8 to 3.1.9-beta1
2. Try to open the client
3. Blank white screen with nothing going on

### Screenshots

![image](https://user-images.githubusercontent.com/70153520/220413662-9c87601b-f3f3-4619-a111-916da564221f.png)

**Comment by b5414** [comment]
Same on windows 10, github app version - `3.1.9-beta1`

After closing, it stops opening, 4 tasks hang in the task manager

**Comment by Buffele** [comment]
Also encountering the same issue.

This is the error in the developer console:

```
Uncaught TypeError: Cannot convert undefined or null to object
    at index.tsx:372:18
    at Array.forEach (<anonymous>)
    at index.tsx:370:36
    at index.tsx:369:2
    at index.tsx:387:27
    at index.tsx:387:27
```

**Comment by niik** [comment]
Thank you all for letting us know about this! Turns out the beta release was missing #16151 which fixed a crash that didn't manifest itself in development builds. We've pulled the bad release from the beta channel and will be releasing [3.1.9-beta2 soon](https://github.com/desktop/desktop/pull/16194) to replace it.

In the meantime you can manually download the last stable release using the links in the README (pasted below for convenience) and replace your existing app with it.

 - [macOS](https://central.github.com/deployments/desktop/desktop/latest/darwin?env=beta)
 - [macOS (Apple silicon)](https://central.github.com/deployments/desktop/desktop/latest/darwin-arm64?env=beta)
 - [Windows](https://central.github.com/deployments/desktop/desktop/latest/win32?env=beta)
 - [Windows (ARM64)](https://central.github.com/deployments/desktop/desktop/latest/win32-arm64?env=beta)

**Comment by niik** [comment]
We've just shipped [3.1.9-beta2](https://github.com/desktop/desktop/pull/16194) which fixes this issue. You can download it from the links posted in my previous comment. Thank you for your patience as we worked to resolve this and our apologies for the friction it caused! We appreciate your help testing out our beta releases ❤️
